.github/workflows/test-package.yml

```diff
@@ -91,19 +117,29 @@ jobs:
             exit 1
           fi
 
+      - name: List installed Stylelint versions
+        working-directory: ${{ steps.repo-info.outputs.directory }}
+        env:
+          PACKAGE_URL: ${{ steps.repo-info.outputs.url }}
+          LIST_INSTALLED_VERSIONS_COMMAND: ${{ inputs.list-installed-versions-command }}
+        run: |
+          if ! eval "$LIST_INSTALLED_VERSIONS_COMMAND"; then
+            echo "::error ::Listing dependencies failed."
+            exit 1
+          fi
```

Comment:
This allows us to get some feedback on what version is actually installed by these package managers.

```
"resolved": "https://codeload.github.com/stylelint/stylelint/tar.gz/860acdccb666ee3429d8509e596ce81708448b3f",
```

vs.

```
"resolved": "https://registry.npmjs.org/stylelint/-/stylelint-16.8.2.tgz",
```

For `pnpm` the `json` format for the `list` command gives the most readable output.
The plain text format is too dense, too much in one long sentences, making it harder to scan and check if the installed version is actually different.